sass/compat/themes/leadengine/_checkout-steps.scss

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------
+// CHECKOUT STEPS
+// ------------------------------------
+
+body#{ $_body-theme-selector } .fc-wrapper .tt_button {
+	z-index: z( 'default' );
+}
+
+body#{ $_body-theme-selector } .fc-wrapper .tt_button.fc-place-order-button{
+	-webkit-font-smoothing: auto;
```

Comment:
@luiggidev,
- Setting the font-smoothing property is usually not needed. Why is it necessary for this theme? Add a comment.
- Avoid using browser specific perperties, usually starting with `-` such as `-webkit` or `-moz`. Use the respective official property as the build process already adds the browser prefixed properties based on browser support: https://caniuse.com/?search=font-smoothing
- For this property specifically there is no official property name as noted in the caniuse website.
- In this case, and if this code is indeed needed, please add an inline comment explaining why the prefixed property name is needed.